### PR TITLE
Make Icons thinner to look more at home with newer firefox versions & add linux build script

### PR DESCRIPTION
**Pull request by fischer-felix**
Hi, I made all the icons a bit thinner, as the originals looked quite chunky on the newer versions of Firefox.
I also added a Linux shell script that does the exact same as the batch script for anyone using *nix like systems.

### CODE REVIEW

**Comment by trubens** [comment]
Thanks for the PR! It definitely looks a lot better than the current one! My only concern is that it looks a bit different than Firefox' own reload icon (see attachment). Maybe we should try to track down that icon?

image

**Comment by fischer-felix** [comment]
I managed to find the original icons [here](https://searchfox.org/mozilla-central/source/browser/themes/shared/icons) (stop-to-reload.svg & reload-to-stop.svg). I copied the stop and reload icons out (possibly in the future we could look at recreating the animation) and replaced the old ones.
HOWEVER: the reload icon looks great, but imo the stop icon looks a bit thick:
Screenshot_2021 12 10_21:54:45
(on the right is just the default reload button)

Do you have any idea why this could be, because the x/y dimensions seem to fit?

File: src/icons/.directory [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[Dolphin]
```
Comment (trubens) [inline src/icons/.directory:1]: Please delete this file

File: src/icons/oldish/refresh-16-dark.svg [hunk 2]
```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (trubens) [inline src/icons/oldish/refresh-16-dark.svg]: We don't need the old icons, so you can delete this folder. It's safe in git :)

File: stop-to-reload.svg [hunk 3]
```diff
@@ -0,0 +1,173 @@
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
```
Comment (trubens) [inline stop-to-reload.svg]: We don't need this file

File: reload-to-stop.svg [hunk 4]
```diff
@@ -0,0 +1,591 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (trubens) [inline reload-to-stop.svg]: We don't need this file

**Comment by trubens** [comment]
I think it looks good, so if you just clean up unnecessary files I'll merge it :)

**Comment by fischer-felix** [comment]
should be clean now

**Review by trubens — APPROVED** [review]

**Comment by trubens** [comment]
Thanks :)

**Pull request merged**